pom.xml

```diff
@@ -1,66 +1,95 @@
 <?xml version="1.0"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
 
-	<groupId>com.esri.geoevent.parent</groupId>
-	<artifactId>track-idle-detector</artifactId>
-	<version>10.6.0</version>
-	<packaging>pom</packaging>
+  <groupId>com.esri.geoevent.parent</groupId>
+  <artifactId>trackidledetector</artifactId>
```

Comment:
We should not change the artifact id

Reply:
Needs to match the path to the resources string folder/file. Otherwise strings don't load nor get replaced by the resources bundle.